qiskit_nature/problems/sampling/sampling_qubit_op_builder.py

```diff
@@ -0,0 +1,25 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2021.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+"""An interface for sampling qubit operator builders."""
+from abc import ABC, abstractmethod
+from typing import Union
+
+from qiskit.opflow import PauliOp, PauliSumOp
+
+
+class SamplingQubitOpBuilder(ABC):
```

Comment:
Since #220 is replacing most other operator builders with generic properties, I'm wondering whether we will be able to do something similar here.
This is not a suggestion for now but rather an FYI. I'm open for discussion/ideas :slightly_smiling_face:

Reply:
Sure, thanks. We will think about it.